task/eaas-provision-space/0.1/eaas-provision-space.yaml

```diff
@@ -58,27 +60,26 @@ spec:
             ;;
         esac
 
-        cat <<EOF > space_request.yaml
-        apiVersion: toolchain.dev.openshift.com/v1alpha1
-        kind: SpaceRequest
+        cat <<EOF > namespace_claim.yaml
+        apiVersion: eaas.konflux-ci.dev/v1alpha1
+        kind: Namespace
         metadata:
-          generateName: eaas-spacerequest-
+          name: $SECRET
```

Comment:
This variable, `$SECRET`, is misleading. It's not the name of the Secret as you'd expect. It's the name of the TaskRun. Consider adding another env variable for the name of the claim while fixing `$SECRET`.